apps/editor/src/app/routes/imageList.tsx

```diff
@@ -47,6 +60,81 @@ const Img = styled.img`
   display: block;
   margin: 0 auto;
 `
+const ButtonGroup = styled(RButtonGroup)`
+  margin-top: 10px;
+`
+
+const GridImg = styled.img`
+  height: 140px;
+  width: auto;
+  display: block;
+  margin: 0 auto;
+`
+
+const ImgDesc = styled.p`
+  position: absolute;
+  bottom: 10px;
+  width: 100%;
+  text-align: center;
+  font-size: 12px;
+  text-shadow: 1px 1px white;
+  display: inline-block;
+  background: white;
+  padding: 2px;
+`
+const GridIcon = styled(IconButton)`
+  position: absolute;
+  top: 5px;
+  right: 5px;
+`
+
+const GridView = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  row-gap: 20px;
+  margin: 20px 0;
+`
+
+const ImageWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 20%;
```

Comment:
nit: If we specify hard widths like these, we should use grid and not flex.
It's fine to keep it like this though, as the benefit now is not worth to spend more time on it.